pkg/init/Dockerfile

```diff
@@ -1,9 +1,20 @@
-FROM alpine:3.5
+FROM linuxkit/alpine:f0169b60fb260d74025496ae6fd93213fecaba8f@sha256:23743c7206ebe8a609442c5ac7084a26ed45ce8f5213960428bca264225849f1 AS build
+# XXX There is an issue with the bsd-compat-headers package I don't
+# understand. Installing the file directly is yuk but works
+RUN apk --no-cache add \
+    /mirror/x86_64/bsd-compat-headers-*.apk \
+    gcc \
+    linux-headers \
+    musl-dev
+COPY ./start-stop-daemon.c .
 
-RUN \
-  apk --no-cache update && \
-  apk --no-cache upgrade -a && \
-  apk --no-cache add \
-  && rm -rf /var/cache/apk/*
+RUN cc -static -O2 -Wall -Werror -o start-stop-daemon start-stop-daemon.c
+
+# Pull in the latest 3.5 so we don't have to update/upgrade
+FROM alpine:3.5@sha256:58e1a1bb75db1b5a24a462dd5e2915277ea06438c3f105138f97eb53149673c4
```

Comment:
this should use `edge` as that is what the linuxkit/alpine package is built from now (we may move to 3.6 when it is out).